packages/core/src/device/index.ts

```diff
@@ -1,28 +1,25 @@
-export type DeviceModelId = string;
-
-export type Device = {
-  [cws: string]: DeviceModel
-}
+export type BluetoothSpec = {
+  serviceUuid: string;
+  writeUuid: string;
+  dataUuid: string;
+  checkUuid: string;
+  readUuid: string;
+};
 
 export type DeviceModel = {
-  id: string,
-  productName: string,
-  bluetoothSpec?: Array<BluetoothSpec>
+  id: string;
+  productName: string;
+  bluetoothSpec?: Array<BluetoothSpec>;
 };
 
-export type BluetoothSpec = {
-  serviceUuid: string,
-  writeUuid: string,
-  dataUuid: string,
-  checkUuid: string,
-  readUuid: string
-}
+export type Device = {
+  [cws: string]: DeviceModel;
```

Comment:
This SDK does not support CoolWallet S, so I'm considering `cws` is not proper here. And `productName` shall be `CoolWallet Pro` or `CoolWallet`.